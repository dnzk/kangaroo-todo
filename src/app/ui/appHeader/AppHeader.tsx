import style from "./appHeader.module.scss"
import AppLogo from "../logo/AppLogo"
import profile from "@public/profile.svg"
import Image from "next/image"

export default function AppHeader() {
  return (
    <div className={style['app-header']}>
      <h1><AppLogo variant="medium" /></h1>
      <Image src={profile} alt="Profile" />
    </div>
  )
}
