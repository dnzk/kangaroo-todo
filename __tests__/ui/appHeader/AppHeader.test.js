import '@testing-library/jest-dom'
import { render, screen } from '@testing-library/react'
import AppHeader from '../../../src/app/ui/appHeader/AppHeader'

describe('AppHeader', () => {
  it('renders correctly', () => {
    const { container } = render(<AppHeader />)
    expect(container).toMatchSnapshot()
  })

  it('should have AppLogo', () => {
    render(<AppHeader />)

    const logo = screen.getByAltText(/Kangaroo Todo/i)
    expect(logo).toBeInTheDocument()
  })
})
