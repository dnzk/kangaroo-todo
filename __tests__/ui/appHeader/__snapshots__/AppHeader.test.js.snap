// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppHeader renders correctly 1`] = `
<div>
  <div
    class="app-header"
  >
    <h1>
      <img
        alt="Kangaroo Todo"
        data-nimg="1"
        decoding="async"
        height="113"
        loading="lazy"
        src="/_next/image?url=%2Fimg.jpg&w=256&q=75"
        srcset="/_next/image?url=%2Fimg.jpg&w=128&q=75 1x, /_next/image?url=%2Fimg.jpg&w=256&q=75 2x"
        style="color: transparent;"
        width="113"
      />
    </h1>
    <img
      alt="Profile"
      data-nimg="1"
      decoding="async"
      height="40"
      loading="lazy"
      src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
      srcset="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
      style="color: transparent;"
      width="40"
    />
  </div>
</div>
`;
